File: EffortEstimator/Models/EstimationForm.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace EffortEstimator.Models
{
    public class EstimationForm
    {
        public string Topic { get; set; }
        public string Description { get; set; }
        public int Iteration { get; set; }
        public double UserResultValue { get; set; }
        public double MinResultValue { get; set; }
        public double MaxResultValue { get; set; }
        public double AvgResultValue { get; set; }
        public double ProposedValue { get; set; }
        public List<double> ResultValues { get; set; }
    }
}

File: EffortEstimator/ClientApp/src/app/objects/estimation-form.ts
export class EstimationForm {
    topic: string;
    description: string;
    iteration: number;
    userResultValue: number;
    minResultValue: number;
    maxResultValue: number;
    avgResultValue: number;
    proposedValue: number;
    resultValues: number[];

    constructor(topic: string, description: string, iteration: number, userResultValue: number, minResultValue: number,
        maxResultValue: number, avgResultValue: number, proposedValue: number, resultValues: number[]) {
        this.topic = topic;
        this.description = description;
        this.iteration = iteration;
        this.userResultValue = userResultValue;
        this.minResultValue = minResultValue;
        this.maxResultValue = maxResultValue;
        this.avgResultValue = avgResultValue;
        this.proposedValue = proposedValue;
        this.resultValues = resultValues;
    }
}
